setup.sh

```diff
@@ -26,6 +26,9 @@ clean(){
 install(){
   echo 'Installing project dependencies...'
   npm install
+  echo 'Shrinkwrapping project dependencies...'
+  npm prune
+  npm shrinkwrap --dev
```

Comment:
It depends on your build flow but you probably don't want to run `npm shrinkwrap` on every install. It's generally only used when you've updated one or more dependencies and want to regenerate your `npm-shrinkwrap.json` file.